diff --git a/android/src/main/java/org/godotengine/plugin/admob/AdmobPlugin.java b/android/src/main/java/org/godotengine/plugin/admob/AdmobPlugin.java
index a9be132..f8a625e 100644
--- a/android/src/main/java/org/godotengine/plugin/admob/AdmobPlugin.java
+++ b/android/src/main/java/org/godotengine/plugin/admob/AdmobPlugin.java
@@ -125,0 +126 @@ public class AdmobPlugin extends GodotPlugin {
+	static final String SIGNAL_PRIVACY_OPTIONS_FORM_DISMISSED = "privacy_options_form_dismissed";
@@ -259,0 +261 @@ public class AdmobPlugin extends GodotPlugin {
+		signals.add(new SignalInfo(SIGNAL_PRIVACY_OPTIONS_FORM_DISMISSED, Dictionary.class));
@@ -1002,0 +1005,18 @@ public class AdmobPlugin extends GodotPlugin {
+	@UsedByGodot
+	public String get_privacy_options_requirement_status() {
+		ConsentInformation.PrivacyOptionsRequirementStatus status =
+				UserMessagingPlatform.getConsentInformation(activity).getPrivacyOptionsRequirementStatus();
+		Log.d(LOG_TAG, String.format("get_privacy_options_requirement_status(): %s", status));
+		return status.name();
+	}
+
+	@UsedByGodot
+	public void show_privacy_options_form() {
+		Log.d(LOG_TAG, "show_privacy_options_form()");
+		activity.runOnUiThread(() -> UserMessagingPlatform.showPrivacyOptionsForm(
+				activity,
+				(ConsentForm.OnConsentFormDismissedListener) formError ->
+						emitSignal(SIGNAL_PRIVACY_OPTIONS_FORM_DISMISSED,
+								GodotConverter.convert(formError))));
+	}
+
